store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -58,11 +58,20 @@ public abstract class AbstractCoreKeyAccumuloElementConverter implements Accumul
     protected final Schema schema;
     private final String timestampProperty;
     private final Set<String> aggregatedGroups;
+    private final Set<String> timeSensitiveAggregatedGroups = new HashSet<>();
 
-    public AbstractCoreKeyAccumuloElementConverter(final Schema schema) {
```

Comment:
Although this seems sensible, I would maybe leave it public as there might be users using this class directly.

Reply:
The class is abstract users cant instantiate it directly anyways so it would be a code smell to leave it public.

Reply:
Oops good point didn't catch that...